src/app/home/home.component.scss

```diff
@@ -76,3 +76,87 @@ span {
   margin-bottom: 2px;
   font-weight: 700;
 }
+
+.button-link {
+  text-decoration: none !important;
+  color: black !important;
+}
+
+.landing-search {
+  width: 60%;
+  max-width: 600px;
+  padding-left: 30px;
+  padding-top: 3px;
+  padding-bottom: 3px;
+  border: none;
+  line-height: 2em;
+  background: white url(../../../images/search-button.png) left center no-repeat;
+  background-position: 5px;
+  outline: none;
+  color: black;
+}
+
+.search-title {
+  color: white;
+  margin: 20px 0px 0px 0px;
+}
+
+.sign-up, .sign-up:active, .sign-up:focus {
+  float: none;
+  margin-top: 30px;
+  margin-bottom: 30px;
+  background: none;
+  text-transform: uppercase;
+  padding: 8px 22px;
+
+  border: 2px solid #5e7ec5;
+  color: #5e7ec5;
+}
+
+.sign-up:hover {
+  border: 2px solid #3b5da5;
+  color: #3b5da5;
+}
+
+.top-section {
+  min-height: 800px;
+}
+
+.search-holder {
+  margin-top: 40px;
+}
+
+.header-lower {
+  min-height: 200px;
+}
+
+#developerIntroduction {
+  margin-top:40px;
+}
+
+.developer-holder {
+  padding: 40px 0px 40px 0px;
+  min-height: 400px;
+}
+
+.stripe-holder {
+  min-height: 175px;
+  margin: 40px 0px;
+  width: 100%;
+  background-color: #eef0f6;
+  border: 1px solid #cfd1d7;
+}
+
+@media (max-width: 767px) {
+  .hidden-on-mobile {
+     display: none !important;
+  }
+
+  .flex-column-on-mobile {
+    flex-direction: column !important;
+  }
+}
+
+.youtube {
+  z-index: 1000;
```

Comment:
Use https://getbootstrap.com/docs/4.3/layout/overview/#z-index values.  Put the new class above
```
.bootstrap-tooltip-z-index {
  z-index: 1070;

}
```